starter-code/css/main.css

```diff
@@ -1,8 +1,197 @@
 /*
 Colors:
 
-Text: 1A1A1A
+Text: #1A1A1A
 Green: #00B172
 White: #FFF
 
 */
+* {
+    margin: 0;
+    padding: 0;
+}
+
+
+.header {
+    height: 100vh;
+    width:100%;
+    position: relative;
+    background-image: url(../images/landing.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+.nav-bar {
+    display: flex;
+    background-color: white;
+    width: 100%;
+    height: 60px;
+    justify-content: space-between;
+    position: fixed;
+}
+
+.nav-logo {
+    display:flex;
+    align-items: center;
+}
+
+.nav-logo img {
+    width: 120px;
+    height: auto;
+    margin-left: 10px;
+}
+
+.nav-bar ul {
+    display: flex;
+    align-items: center;
+}
+
+.nav-bar li {
+    display: flex;
+    list-style-type: none;
+    padding: 8px;
+    font-size: 10px;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.nav-bar a {
+    color: black;
+    text-decoration: none;
+    font-size: 1.5em;
+}
+
+
+.header-text {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.header-text h1 {
+    font-size: 50px;
+}
+.header-text h1, p {
+    text-align: center;
+    color: white;
+    font-family: Arial, Helvetica, sans-serif;
+
+}
+
+
+/*  FIN SECTION 1   */
+
+
+.mid-section {
+    text-align: center;
+    padding: 40px;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.mid-section hr {
+    width: 210px;
+    margin:auto;
+    margin-top: 5px;
+    padding: 1px;
+    background-color: #00B172 ;
+}
+
+.article {
+    margin: 40px;
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.info-box {
+    width: 300px;
+    height: 250px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.info-box img {
+    width: 125px;
+    margin-top: 15px;
+}
+
+.info-box h3 {
+    color: #00B172;
+    margin-top: 5px;
+    margin-bottom: 10px;
+}
+
+
+.info-box p {
+    color: #1A1A1A;
+    font-size: smaller;
+    box-sizing: border-box;
+}
+
+
+/*   FIN SECTION 2   */
+
+
+.footer {
+    background-color: #00b172;
+    display: flex;
+    padding: 50px;
+    margin: 20px;
+    position: static;
+    justify-content: space-between;
+}
+
+#white-logo {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+}
+
+#white-logo img {
+    width: auto;
+    height: 90px;
+}
+
+#app-screen {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+#app-screen img {
+    width: auto;
+    height: 350px;
+
+}
+
+.last-section {
+    color: white;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.last-section hr {
+    width: 300px;
+    margin-top: 5px;
+    padding: 1px;
+    background-color: white;
+}
+
+.last-section h2 {
+    font-weight: lighter;
+}
+
+.last-section h3 {
+    margin-bottom: 40px;
+    margin-top: 40px;
+}
+
+
+.last-section p {
+    font-weight: lighter;
+    width: 200px;
+    text-align: left;
+}
```

Comment:
You might know it already, but commenting just in case.

These selectors are "dangerous" in a way that if you add an element to the last section that is not an article, these styles will be applied to it.
You could add a class to the article and access  the h3 through the article class.